src/carts/README.md

```diff
@@ -1,40 +1,130 @@
+
+    * **Analytics** - Oracle Database is the world's leading [translytic database](https://blogs.oracle.com/database/oracle-1-in-forresters-translytical-data-platforms-wave-v2).  Even though the cart microservice is written without using SQL, SQL can still be used to access JSON collections.  The data can be exposed, in-place, to existing analytics tools that don't necessarily support JSON and might use older database drivers.  There are no special restrictions on the types of queries that can be used over JSON collections.  In contrast, NoSQL databases typically have significant restrictions on the types of joins and subqueries that can be expressed and don't support standard SQL drivers such as JDBC.
+
+      ```SQL
+      SELECT c.json_document.customerId
+      FROM carts c
+      ```
+      See [examples.sql](examples.sql) for more examples of how SQL can be used over the _carts_ collection used by this service.
+
+    * **Multimodel** - Data stored in JSON collections can be queried along side other types of data in Oracle Database such as relational, geospatial, graph, and so on.  See [examples.sql](examples.sql) for an example that joins the _cart_ collection with other relational tables.
+
+    * **Autonomous** - JSON collections benefit from all the general features of the [autonomous database](https://www.oracle.com/database/what-is-autonomous-database.html) such as advanced security, automated patching, automated backups, and so on.
 
-`GROUP=aurademos/durhamdenim COMMIT=test ./scripts/build.sh`
+    * **Cost** - JSON collections are in the [always-free tier](https://www.oracle.com/cloud/free/) of the autonomous database.  You can run this shopping cart service for free, forever, in the Oracle Cloud.
 
-# Test
+  * **Helidon** - The REST services are built using [Helidon SE](https://helidon.io/), a lightweight, fast platform for building microservices in Java.  Helidon makes it easy to map REST URL patterns to Java functions.
 
-`./test/test.sh < python testing file >`. For example: `./test/test.sh unit.py`
+    ```java
+     public void update(Rules rules) {
+        rules.get("/{cartId}/items", this::getCartItems);
+     }
 
-# Run
+     public void getCartItems(ServerRequest request, ServerResponse response) {
+        String cartId = request.path().param("cartId");
+        result = carts.getById(cartId);
+        response.status(200).send(result.getItems());
+     }
+     ```
+     See [src/main/java/mushop/carts/CartService.java](src/main/java/mushop/carts/CartService.java)
 
-`mvn spring-boot:run`
+     Helidon is designed to support cloud-native applications. It comes with built-in support for things like health checks, metrics, tracing, and fault tolerance.  These features make it work well for deployment in Docker and Kubernetes.
```

Comment:
Do you have `/metrics`, `/health` enabled? Ideally `/health` would also reflect db connectivity check.

Reply:
Added metrics with corresponding unit test, as discussed.